src/arrayMethodReduce.js

```diff
@@ -6,6 +6,19 @@
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
     // write code here
+    let accumValue = initialValue;
+    let firstIndex = 0;
+
+    if (initialValue == null) {
```

Comment:
```suggestion
    if (initialValue === undefined) {
```
Насколько я знаю, лучше всегда использовать строгое сравнение

Reply:
Да, спасибо большое, я исправил) В данном случае так будет лучше, так как initialValue не ожидается null.
Но в целом, нестрогое сравнение с null,  сравнивает с null и undefined.